docs/pbc/all/dynamic-multistore/202311.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -80,3 +80,58 @@ Normal storefront APIs can be used to fetch data from stores using the Store HTT
 The following image shows the ability to call API endpoints with the exemplary "SECOND" store selected in the API call itself.
 
 ![storefront-api](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/storefront-api.png)
+
+## Adding of a new store via data import
+
+Example for DE store configuration:
+
+**data/import/common/{REGION}/store.csv**
+
+```csv
+name
+DE
+AT
+```
+
+| Column     | REQUIRED | Data Type | Data Example | Data Explanation |
+|------------| --- | --- | --- | --- |
+|name        |mandatory |string | DE | Define store name. |
+
+Configure a new store as described here: [Import data for store](/docs/pbc/all/dynamic-multistore/202311.0/base-shop/install-and-upgrade/install-features/install-dynamic-multistore.html#import-data)
+
+
+Adding of a new store via Back Office is easier and faster way but in this case you should add this store at all environments. 
+Data import allows you to add and configure a new store once and deploy it to all environments. 
+
+Note that when you add a new store you should also assign all store-related entities to it for enabling them for customers:
+- Products
+- Categories
+- CMS entities
+- Prices. See [Install Dynamic Multistore + the Prices feature](/docs/pbc/all/dynamic-multistore/202311.0/base-shop/install-and-upgrade/install-features/install-dynamic-multistore-prices-feature.html#install-feature-core)
+- etc.
+
```

Comment:
Do we have anywhere described the **minimal** scope of all needed setups to make a new store alive for Backoffice and data import? I mean if I were a customer it would be beneficial to have a doc with all the steps that guarantee I follow it and have a live environment with products. For example, assign the category, assign the warehouse and that's all that you need without `etc` so I don't have to guess what is missed.
If yes I would add a link here otherwise to describe both for the Back Office and Data import. 

@asmarovydlo @gechetspr what do you think?

Reply:
Created a page with minimal set of data for import